packages/kolibri-common/composables/useFacilities.js

```diff
@@ -0,0 +1,82 @@
+import { reactive, computed } from 'vue';
+import FacilityResource from 'kolibri-common/apiResources/FacilityResource';
+import useUser from 'kolibri/composables/useUser';
+import redirectBrowser from 'kolibri/utils/redirectBrowser';
+import FacilityDatasetResource from 'kolibri-common/apiResources/FacilityDatasetResource';
+import Lockr from 'lockr';
+
+const state = reactive({
+  facilityConfig: {},
+  facilities: [],
+  facilityId: Lockr.get('facilityId') || null,
```

Comment:
Have not removed the existing facilitiyId state just added this here to keep a local copy of facilityId.

Reply:
Hrm, but we're not actually setting this anywhere either. Let's remove this, it's not in the original, so not sure that we need it, and if we're not setting it, it's not doing anything anyway.